test/new-e2e/pkg/utils/infra/stack_manager.go

```diff
@@ -250,11 +248,23 @@ func (sm *StackManager) getStack(ctx context.Context, name string, config runner
 		logger = logWriter
 	}
 
-	upResult, err := stack.Up(upCtx, optup.ProgressStreams(logger), optup.DebugLogging(debug.LoggingOptions{
-		FlowToPlugins: true,
-		LogLevel:      &loglevel,
-	}))
-
+	var ok bool
+	var upResult auto.UpResult
+
+	for retry := 0; retry < 2 && !ok; retry++ {
+		upCtx, cancel := context.WithTimeout(ctx, stackUpTimeout)
+		upResult, err = stack.Up(upCtx, optup.ProgressStreams(logger), optup.DebugLogging(debug.LoggingOptions{
+			FlowToPlugins: true,
+			LogLevel:      &loglevel,
+		}))
+		cancel()
+		if err != nil && shouldRetryError(err) {
+			fmt.Fprintf(logger, "Got error that should be retried during stack up, retrying: %v\n", err)
+			time.Sleep(5 * time.Second)
```

Comment:
Given the time it takes to restart the provisioning, I don't think this is necessary.